build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -452,10 +452,10 @@ service Capabilities {
 // reproducible so that serving a result from cache is always desirable and
 // correct.
 message Action {
-  // The digest of the [Command][build.bazel.remote.execution.v2.Command]
+  // The digest and optionally content of the [Command][build.bazel.remote.execution.v2.Command]
   // to run, which MUST be present in the
   // [ContentAddressableStorage][build.bazel.remote.execution.v2.ContentAddressableStorage].
-  Digest command_digest = 1;
+  Blob command = 1;
```

Comment:
I strongly object to this change.

With the platform properties being promoted from Command to Action, scheduler processes may nowadays be implemented in such a way that they don't need to load the Command message from the CAS. This effectively reverts that change.

The Command message can get big really quickly, especially in the case of projects where people use (strip_)include_prefix too much.

Reply:
To add to that: is it even likely that inlining Command into Action is advantageous? You only upload Action and Command into the CAS in case of remote execution; not remote caching. Most remote execution services hopefully have their workers located close to storage. Do we really care about reducing latency/roundtrips there?

Reply:
Sounds reasonable- this is the "over-inlining" problem I mentioned in the description (this is only intended as an early draft for discussion).